File: app/src/main/java/com/evaluation/model/result/location/coordinates/Coordinates.java
package com.evaluation.model.result.location.coordinates;

import com.google.gson.annotations.SerializedName;

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
public class Coordinates {

    @SerializedName("latitude")
    private String mLatitude;

    @SerializedName("longitude")
    private String mLongitude;

    public String getLatitude() {
        return mLatitude;
    }

    public String getLongitude() {
        return mLongitude;
    }
}

File: app/src/main/java/com/evaluation/model/result/location/coordinates/Coordinates.kt
package com.evaluation.model.result.location.coordinates

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
data class Coordinates(
    @SerializedName("latitude")
    val mLatitude: String,

    @SerializedName("longitude")
    val mLongitude: String
)